features/util.go

```diff
@@ -166,6 +192,20 @@ func (b *featureGateBuilder) register() (configv1.FeatureGateName, error) {
 			} else {
 				allFeatureGates[clusterProfile][featureSet].Disabled = append(allFeatureGates[clusterProfile][featureSet].Disabled, description)
 			}
+			if b.minimumKubeletVersion != "" && featureSet == configv1.Default {
```

Comment:
Does minimumKubeletVersion only work with the default feature set? Won't work in tech preview?

Reply:
correct I think it should only apply to the default. I think we'd go Custom->TechPreview->minimumKubeletVersion (optional)->GA

Reply:
Huh, I had imagined they would be orthogonal 🤔 

Does that mean moving to a minimumKubeletVersion counts as promotion of the feature to the stable feature set? Or can the addition of the minimum kubelet be done without the feature being in the stable feature set?

Reply:
>Does that mean moving to a minimumKubeletVersion counts as promotion of the feature to the stable feature set? 

I am thinking of it this way, for user namespaces at least.